src/test/ui/closures/2229_closure_analysis/diagnostics/borrowck/borrowck-closures-mut-and-imm.stderr

```diff
@@ -2,7 +2,7 @@ error[E0502]: cannot borrow `p` as mutable because it is also borrowed as immuta
   --> $DIR/borrowck-closures-mut-and-imm.rs:17:14
    |
 LL |     let c2 = || p.y * 5;
-   |              -- --- first borrow occurs due to use of `p.y` in closure
+   |              -- --- first borrow occurs due to use of `p` in closure
```

Comment:
I'm pretty sure it's a bug here so I blessed, but I wonder if this the case can be fixed with `eager`?

Reply:
Worth a try?

Reply:
oh I managed to find that handler!